grass7/raster/r.in.usgs/r.in.usgs.py

```diff
@@ -684,6 +740,10 @@ def exist_list():
                             removed=removed_extracted_tiles_num
                             ))
 
+    if gui_product == 'lidar' and not has_pdal:
```

Comment:
Same condition and error as above? Needed? (also missing a space)

Reply:
First is for warning, still allows to download lidar, the second one is fatal to stop processing.